lib/dbaccessor_test.go

```diff
@@ -287,14 +287,63 @@ func testInsertAndGetUser(ta TestAccessor, t *testing.T) {
 func testInsertAndGetFilteredUsers(ta TestAccessor, t *testing.T) {
 	t.Log("TestInsertAndGetFilteredUsers")
 	ta.Truncate()
+	// create user
+	insertTypesClient := cadbuser.Info{
+		Name: "testTypesClient",
+		Pass: "123456",
+		Type: "client",
+		Attributes: []api.Attribute{
+			api.Attribute{
+				Name:  "hf.Registrar.Roles",
+				Value: "peer,client,orderer,user",
+			},
+			api.Attribute{
+				Name:  "hf.Revoker",
+				Value: "false",
+			},
+			api.Attribute{
+				Name:  "hf.Registrar.Attributes",
+				Value: "*",
+			},
+			api.Attribute{
+				Name:  "xyz",
+				Value: "xyz",
+			},
+		},
+	}
+	// insert user
+	err := ta.Accessor.InsertUser(&insertTypesClient)
```

Comment:
Errors can't be ignored; this needs an assertion to ensure an error did not occur.

Reply:
I have handled the error.